modules/RTC/ortc/RTCPeerConnection.js

```diff
@@ -703,6 +1900,7 @@ export default class ortcRTCPeerConnection extends yaeti.EventTarget {
 
     /**
      * May emit 'negotiationneeded' event.
+     * @private
```

Comment:
No please, don't ask me to do that. This is about reading code in a logical way. Would you place an `addItem()` method **before** the `constructor()`?